irispy/iris_tools.py

```diff
@@ -504,8 +504,7 @@ def _calculate_orbital_wavelength_variation(data_array, date_data_created, slit_
             # Undefined name 'slit_axis' in line below.
             data_array = data_array.isel(slit_axis, slice(slit_pixel_range[0], slit_pixel_range[1]))
         else:
-            raise TypeError("slit_pixel_range must be tuple of length 2 giving lower and " +
-                            "upper bounds of section of slit over which to average line fits.")
+            raise TypeError("slit_pixel_range must be tuple of length 2 giving lower and " + "upper bounds of section of slit over which to average line fits.")
```

Comment:
PEP8 compliance:  Lines must be <100 characters.  This line looks like it might be more than that.  If so, break into two lines as previously.